streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -1337,7 +1337,7 @@ public boolean isOpen() {
         public TestDriverProducer(final StreamsConfig config,
                                   final KafkaClientSupplier clientSupplier,
                                   final LogContext logContext) {
-            super(config, "TopologyTestDriver-StreamThread-1", clientSupplier, new TaskId(0, 0), UUID.randomUUID(), logContext);
+            super(config, "TopologyTestDriver-StreamThread-1", clientSupplier, new TaskId(0, 0), UUID.randomUUID(), logContext, Time.SYSTEM);
```

Comment:
I think this should be:
```suggestion
            super(config, "TopologyTestDriver-StreamThread-1", clientSupplier, new TaskId(0, 0), UUID.randomUUID(), logContext, mockWallClockTime);
```